lib/util/package-rules/current-version.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('util/package-rules/current-version', () => {
     });
 
     it('return false on version exception', () => {
-      const spy = jest.spyOn(pep440, 'matches').mockImplementationOnce(() => {
+      const spy = jest.spyOn(pep440, 'isValid').mockImplementationOnce(() => {
```

Comment:
MOcking a different function, as the `matches` mock was messing with the new test.